src/arena/CreateSprintTemplate.js

```diff
@@ -143,19 +146,108 @@ function CreateSprintTemplate(props) {
     getSanityItems();
   }, []);
 
+  useEffect(() => {
+    getConfiguration();
+  }, []);
+
+  useEffect(() => {
+    handleChange();
+  }, [title, columns, handleChange]);
+
+  // pulls the /templates api and sets the missions to the templates
+  async function getConfiguration() {
+    let options = await RestAPI.get("pareto", "/templates");
+    setMissions(options.map((option) => option.title));
+  }
+  const minimumOptions =
+    columns.Morning.items.length +
+      columns.Workday.items.length +
+      columns.Evening.items.length >=
+    3;
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const handleChange = () => {
+    const templates = missions.filter((mission) => mission === title.trim());
+    let message;
+    if (
+      title === "" &&
```

Comment:
I think this got overwritten somehow in the cleanup process (the way it's written, there's still that edge case where the title field goes back to 'green' if a user deletes the title but still has achievements dragged over to columns.)

The columns count logic (looking back at our Discord troubleshooting) needs to be either 
```
(columns.Morning.items.length === 0 &&
  columns.Workday.items.length === 0 &&
  columns.Evening.items.length === 0)
```
or 
```
(columns.Morning.items.length +
  columns.Workday.items.length +
  columns.Evening.items.length === 0)
```
(Either one will test if all three columns are empty, which is the only situation where an empty title input field should have the error message be "".)
   
![bug](https://user-images.githubusercontent.com/84106309/167885314-96f04ed3-30bb-431c-a5b0-c809d5a23755.gif)

Reply:
Im not sure what happened here, I did put it in, but I was having a bit of trouble merging my code in and had to rebase things several times. Must have messed something up. Ill re-add it in here.